### DISCUSSION TITLE
Is there a place to discuss this game system?

### DISCUSSION THREAD

**Discussion by RandomGiblet** [Mezryss/FVTT-Genesys#45] [Q&A] [answered]
I am having an issue trying to set the default skills compendium. I cannot figure out how to link it to the compendium I created. Thank you.

**Comment by Mezryss** [comment]
Assuming you didn't add the compendium to a module of some kind, it should be typed in as something like `world.my-skill-compendium` (if you named your compendium "My Skill Compendium").

I'm planning at some point to write a wiki properly explaining things, but I haven't had the energy.

**Reply by RandomGiblet** [reply]
I appreciate you getting back to me.  No worries on the wiki. So far this is the only big issue I ran into.

Unfortunately I am still unable to get it working.  I could not get it working in the world I just called "Genesys".  Thinking maybe my world name was conflicting with something, I created a new world without any modules loaded.  I named the new world "twilight" and named the compendium "skills2".  I duplicated the default skill compendium and added one new skill to the compendium just to keep the test as simple as possible.

In the Skills Compendium setting I entered "twilight.skills2" but when I create a new character it tells me "Unable to load Skills Compendium 'twilight.skills2'. Actor was created with no initial skill data. Reconfigure the skills compendium in Genesys Settings."

Let me know if I can get you any additional information to try and figure out what I may be missing.  Thank you.

**Reply by Mezryss** [reply, marked as answer]
You need to use `world.skills2`, not `twilight.skills2`.

**Reply by RandomGiblet** [reply]
That was it. The first world I created was named genesys and then I saw the default setting was genesys.crb-skills so my assumption was the world. meant world name not literally world.
Thank you very much!
